ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/HttpConfiguration.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
```suggestion
 * Copyright contributors to Hyperledger Besu.
```

Reply:
new files should be copyright to besu. no need to update copyright on existing files